docker/demo/config/test-suite/complex-dag-cow.yaml

```diff
@@ -17,23 +17,23 @@ first_insert:
   config:
     record_size: 70000
     num_insert_partitions: 1
-    repeat_count: 5
+    repeat_count: 1
     num_records_insert: 1000
   type: InsertNode
   deps: none
 second_insert:
   config:
     record_size: 70000
     num_insert_partitions: 1
-    repeat_count: 5
+    repeat_count: 1
     num_records_insert: 10000
   deps: first_insert
   type: InsertNode
 third_insert:
   config:
     record_size: 70000
     num_insert_partitions: 1
```

Comment:
@n3nash : all dags need to be fixed for partition config. 
right ones are 
num_partitions_insert
num_partitions_upsert. 
where as so far we are using 
num_insert_partitions
num_upsert_partitions.

Can you fix these and re-run all your tests and dags. specially around upsert and rollbacks to ensure things are looking good.